code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -305,3 +305,14 @@
 		/obj/item/weapon/rrf,
 		/obj/item/weapon/gun/energy/kinetic_accelerator
 		)
+
+/obj/item/weapon/storage/belt/ninja
+	name = "stealth belt"
+	desc = "Can hold a robust array of specialized gear."
+	icon_state = "securitybelt"
+	max_w_class = 4
+	storage_slots = 6
+	can_hold = list(

```

Comment:
This is way too much, a belt that can hold most of the items in the game. Just give them a regulat combat belt or something.

Reply:
Belt will be tweaked to only hold certain items instead of almost everything.